tests/test_annotation_stores.py

```diff
@@ -15,7 +15,7 @@
 import pandas as pd
 import pytest
 from shapely import affinity
-from shapely.geometry import Polygon
+from shapely.geometry import MultiPoint, Polygon
```

Comment:
Can I just check why you are using MultiPoint here? I personally can't see many reasons to use this over multiple Point annotations as it makes spatial queries not work as well. Is this just to test that they do work?